libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.ts

```diff
@@ -17,6 +17,17 @@ export enum PATH_ARROW {
   styleUrls: ['./arrow-variation.component.scss']
 })
 export class ArrowVariationComponent {
+
+  private _printing: boolean;
```

Comment:
Better to define a `false` default value for this property. And since you're using this one in your component logic, make it public and use it in the `ngIf` of the template to keep things consistent.